std/conv.d

```diff
@@ -3113,7 +3112,14 @@ if (isInputRange!Source && isSomeChar!(ElementType!Source) && !is(Source == enum
         }
         else
             static assert(false, "Floating point format of real type not supported");
++/
+        ldval = msdec;
+        if (msscale != 1)               /* if stuff was accumulated in lsdec */
+            ldval = ldval * msscale + lsdec;
```

Comment:
@jpf91, why do all the bit-packing above only for hex strings? Instead, I commented it out and simply assigned the mantissa to the real, as done for decimal strings below. I ignore rounding for now, which is done separately for full 80-bit reals and 64-bit reals above: I'm sure we could add it back in later once we decide how to handle this.